src/main/resources/org/broadinstitute/hellbender/utils/plotter/TangentResultsPlotting.R

```diff
@@ -0,0 +1,168 @@
+# Useful for debugging
+#options(error = quote({dump.frames(dumpto = "CBS_dump", to.file = TRUE); q()}))
+
+library(optparse)
+
+option_list <- list(
+    make_option(c("--sample_name", "-sample_name"), dest="sample_name", action="store"),
+    make_option(c("--targets_file", "-targets_file"), dest="targets_file", action="store"),
+    make_option(c("--pre_tn_file", "-pre_tn_file"), dest="pre_tn_file", action="store"),
+    make_option(c("--seg_file", "-seg_file"), dest="seg_file", action="store"),
+    make_option(c("--output_dir", "-output_dir"), dest="output_dir", action="store"),
+    make_option(c("--log2_input", "-log"), dest="log2_input", action="store"))
+
+opt <- parse_args(OptionParser(option_list=option_list))
+print(opt)
+save(opt, file="debug.RData")
+
+sample_name=opt[["sample_name"]]
+tn_file=opt[["targets_file"]]
+pre_tn_file=opt[["pre_tn_file"]]
+seg_file=opt[["seg_file"]]
+output_file=opt[["output_dir"]]
+log_input=as.logical(opt[["log2_input"]])
+
+# Use a function for debugging purposes
+create_tangent_plots_file = function(sample_name, tn_file, pre_tn_file, seg_file, output_dir, log_input) {
+	# Read in file and extract needed data
+	seg = read.table(seg_file, sep="\t", stringsAsFactors=FALSE, header=TRUE, check.names=FALSE)
+	tn = read.table(tn_file, sep="\t", stringsAsFactors=FALSE, header=TRUE, check.names=FALSE)
+	preTn = read.table(pre_tn_file, sep="\t", stringsAsFactors=FALSE, header=TRUE, check.names=FALSE)
+	# Ability to switch between copy-ratio and log2 copy-ratio
+	if (log_input) {
+	    tnDat = 2^tn[,sample_name]
+	    preTnDat = 2^tn[,sample_name]
+	}else {
+	    tnDat = tn[,sample_name]
+	    preTnDat = tn[,sample_name]
+	}
+    tnContig = tn[,"contig"]
+    tnPos = tn[,"stop"]
+    preTnContig = preTn[,"contig"]
+    preTnPos = preTn[,"stop"]
+    preQc = QC(preTnDat)
+    postQc = QC(tnDat)
+
+    plot.fn = file.path( output_dir, paste(sample_name, "_FullGenome", ".png", sep=""))
+    png(plot.fn, 12, 7, units="in", type="cairo", res=300, bg="white")
+    chr.dat = SetUpPlot("Total copy ratio", 0, 4, "", TRUE)
+    TotalCopyRatioWithSegments(tnDat, tnPos, tnContig, seg, chr.dat)
+    dev.off()
+
+    pre_color="#3B5DFF"
```

Comment:
Add a comment saying that the first color is blue and the second is green or define constants `BLUE` and `GREEN`.